common/test/keyboards/issue6005/HISTORY.md

```diff
@@ -0,0 +1,6 @@
+Vowel Hater Change History
```

Comment:
I think we should avoid the word 'Hater' in the repo. It just doesn't look good... (Can you do a global Search+Replace?)